.gitignore

```diff
@@ -1,5 +1,6 @@
 /.bundle/
 /benchmarks/.bundle/
+/benchmarks/Gemfile.lock
```

Comment:
To be honest I thought we had already done this.

@jemmaissroff @ms-ati am I correct that we should not do this for the root `Gemfile.lock`, so Dependabot can still use those versions? Should we change the root `Gemfile` to pin everything explicitly instead?

Reply:
You know much more about Dependabot than me, but it's my understanding that Dependabot needs the root `Gemfile.lock`, so it would make sense to pin versions right?